packages/template-retail-react-app/app/pages/checkout/partials/shipping-address-selection.jsx

```diff
@@ -105,11 +105,12 @@ const ShippingAddressSelection = ({
     onSubmit = async () => null
 }) => {
     const {formatMessage} = useIntl()
-    const {data: customer} = useCurrentCustomer()
+    const {data: customer, isLoading, isFetching} = useCurrentCustomer()
+    const isLoadingRegisteredCustomer = isLoading && isFetching
```

Comment:
Does this look like we are looking for isInitialLoading? https://tanstack.com/query/v4/docs/react/guides/disabling-queries#isinitialloading

Reply:
Ah yes, it's the same as `isInitialLoading`. Forgot about that.

In the normal cases, I would switch to `isInitialLoading`. But with `useCurrentCustomer`, I feel that it does not really represent what really happens. The query is enabled _only_ if the customer is _registered_. So it's clearer to me if I name it `isLoadingRegisteredCustomer`.